build.sbt

```diff
@@ -79,16 +85,59 @@ libraryDependencies ++= Seq(
   // For reading Neo4j database files.
   "org.neo4j" % "neo4j-community" % "3.5.2",
+  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.10.0",
+)
+
+excludeDependencies ++= Seq(
+  "commons-logging" % "commons-logging",
+  "javax.activation" % "activation",
+  "com.typesafe.akka" % "akka-protobuf-v3_2.12",
+)
+
+assemblyShadeRules in assembly := Seq(
+  ShadeRule.rename("com.typesafe.config.**" -> "lynxkite_shaded.com.typesafe.config.@1").inAll,
+  ShadeRule.rename("com.google.inject.**" -> "lynxkite_shaded.com.google.inject.@1").inAll,
+  ShadeRule.rename("com.google.common.**" -> "lynxkite_shaded.com.google.common.@1").inAll,
+  ShadeRule.rename("com.google.protobuf.**" -> "lynxkite_shaded.com.google.protobuf.@1").inAll,
 )
 
+mainClass in assembly := Some("com.lynxanalytics.biggraph.Main")
+fullClasspath in assembly += Attributed.blank(PlayKeys.playPackageAssets.value)
+assemblyJarName in assembly := s"${name.value}-${version.value}.jar"
+
+assemblyMergeStrategy in assembly := {
+  case PathList("mime.types") => MergeStrategy.concat
+  case PathList("module-info.class") => MergeStrategy.discard
+  case PathList("rootdoc.txt") => MergeStrategy.discard
+  case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
+  case PathList("META-INF", "INDEX.LIST") => MergeStrategy.discard
+  case PathList("META-INF", "DEPENDENCIES") => MergeStrategy.discard
+  case PathList("META-INF", "io.netty.versions.properties") => MergeStrategy.first
+  case x if x.contains("tec/uom/se/format/messages.properties") => MergeStrategy.concat
+  case x if x.contains("META-INF/NOTICE") => MergeStrategy.rename
+  case x if x.contains("META-INF/services") => MergeStrategy.concat
+  case x if x.contains("META-INF/") && x.contains(".DSA") => MergeStrategy.discard
+  case x if x.contains("META-INF/") && x.contains(".RSA") => MergeStrategy.discard
+  case x if x.contains("META-INF/") && x.contains(".SF") => MergeStrategy.discard
+  case x if x.contains("LICENSE") => MergeStrategy.rename
+  case x if x.contains("README") => MergeStrategy.rename
+  case x if x.contains("unused/UnusedStubClass.class") => MergeStrategy.first
+  case x if x.contains("pom.xml") => MergeStrategy.discard
+  case x if x.contains("pom.properties") => MergeStrategy.discard
+  case x if x.contains("reference.conf") => MergeStrategy.concat
+  case x if x.contains("reference-overrides.conf") => MergeStrategy.concat
+  case x => MergeStrategy.deduplicate
+}
+
 // We put the local Spark installation on the classpath for compilation and testing instead of using
 // it from Maven. The version on Maven pulls in an unpredictable (old) version of Hadoop.
 def sparkJars(version: String) = {
   val home = System.getenv("HOME")
-  val jarsDir = new java.io.File(s"$home/spark/spark-$version/jars")
+  val jarsDir = new java.io.File(
+    Option(System.getenv("SPARK_JARS_DIR")).getOrElse(s"$home/spark/spark-$version/jars"))
```

Comment:
Set `SPARK_JARS_DIR` to build against jars harvested from your destination environment.